halo/attest/keeper/valaddr.go

```diff
@@ -0,0 +1,90 @@
+package keeper
+
+import (
+	"context"
+	"sync"
+
+	vtypes "github.com/omni-network/omni/halo/valsync/types"
+	"github.com/omni-network/omni/lib/errors"
+
+	"github.com/cometbft/cometbft/crypto"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// valAddrs is a simple read-through cache for validator comet-to-eth address lookups.
+// It only supports the current validator set; as per cometBFT.
```

Comment:
I'd drop this line -- from my perspective `valAddrs` doesn't care one way or the other about whether or not its entries correspond  to the current validator set. All it knows is the correct mapping of some Comet addrs to Eth addrs.